exosims_by_hand.yml

```diff
@@ -0,0 +1,20 @@
+name: exosims
```

Comment:
We should probably only include the one environment.yml file though I like the simplicity of this one if it contains all the right packages. I just did the simplest thing and let conda export my environment for me for the environment.yml file that's on the main branch, but that probably resulted in a lot of unnecessary clutter.

Reply:
Agree.  I will remove the exosims*.yml files.  I generated these experimental files after I failed to install <emcee> in the <ebs> environment because conda could not solve the environment.  However, these file do not include all the necessary packages.  Going forward, it might be useful to list the necessary dependencies in the repository readme.